### DISCUSSION TITLE
Filter Exports by Delivery Status

### DISCUSSION THREAD

**Discussion by rabbiveesh** [amzn/ads-advanced-tools-docs#254] [Ideas]
As it stands right now (as far as I can tell, at least), the Exports API cannot filter responses based on the calculated status. You can only request to filter for things that are `ENABLED`.

This presents an issue, b/c many advertisers (at least all of my clients) will never disable an ad or a target when they close a campaign. This makes target exports in particular absolutely enormous (I just got one 180MB after gunzipping), most of which is data I'm simply going to skip over.

Could we get a way to pre-filter based on deliveryStatus?

**Comment by elizafarley** [comment]
Thanks for giving us this feedback! I've passed this on to the exports product team to see if we can add this to the roadmap. I'll let you know here if we're able to get a filter added for `deliveryStatus`.
